data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -73,23 +73,65 @@ _ "This attack uses 6 attacks"#enddef
     [/event]
 #enddef
 
-#define WEAPON_SPECIAL_SONG_VERSE ACTIVE_ON X DISTINGUISHER
+#define STR_VERSE_RESTRICT_1
+_ "This attack is the first verse of the Song of Sun Ascension, which progresses by one verse each turn this unit uses a song verse attack. When the last verse is sung or the song sequence is broken due to not attacking for a turn, the song starts again from the first verse."#enddef
+
+#define STR_VERSE_RESTRICT_2
+_ "This attack the second verse of the Song of Sun Ascension, which progresses by one verse each turn this unit uses a song verse attack. When the last verse is sung or the song sequence is broken due to not attacking for a turn, the song starts again from the first verse."#enddef
+
+#define STR_VERSE_RESTRICT_3
+_ "This attack the third verse of the Song of Sun Ascension, which progresses by one verse each turn this unit uses a song verse attack. When the last verse is sung or the song sequence is broken due to not attacking for a turn, the song starts again from the first verse."#enddef
```

Comment:
The second and third verses are missing the words "is the" in "This attack is the nth verse".